Upstream disables libmagic support by default nowadays. For this reason,
the error messages expected in the negative tests also assumes this.
Change the expected error message for magic=1 builds.

This reverts commit 3c1076b31e2e4e3429a5c3d334d555e549fb72d2.

See: https://github.com/derf/feh/commit/3c1076b31e2e4e3429a5c3d334d555e549fb72d2

--- a/test/feh.t
+++ b/test/feh.t
@@ -47,7 +47,7 @@ if ( $version =~ m{ Compile-time \s switches : \s .* help }ox ) {
 }
 
 my $re_warning
-  = qr{${feh_name} WARNING: test/fail/... \- No Imlib2 loader for that file format\n};
+  = qr{${feh_name} WARNING: test/fail/... \- Does not look like an image \(magic bytes missing\)\n};
 my $re_loadable    = qr{test/ok/...};
 my $re_unloadable  = qr{test/fail/...};
 my $re_list_action = qr{test/ok/... 16x16};
